commit dc98cfc6d920e959c7c08a48e4e639bbf447bf48
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Mon Jan 24 02:55:08 2022 +0100

    disable ipx support
    
    this was deprecated in linux and removed in 5.15 headers

diff --git a/pppd/Makefile.linux b/pppd/Makefile.linux
index ab48d50..5bbe8c2 100644
--- a/pppd/Makefile.linux
+++ b/pppd/Makefile.linux
@@ -91,7 +91,7 @@ MAXOCTETS=y
 
 INCLUDE_DIRS= -I../include
 
-COMPILE_FLAGS= -DHAVE_PATHS_H -DIPX_CHANGE -DHAVE_MMAP -pipe
+COMPILE_FLAGS= -DHAVE_PATHS_H -DHAVE_MMAP -pipe
 
 CFLAGS= $(COPTS) $(COMPILE_FLAGS) $(INCLUDE_DIRS) '-DDESTDIR="@DESTDIR@"'
 
